<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.segurosbolivar.automation.tests.login.RunnerLogin" tests="1" skipped="0" failures="1" errors="0" timestamp="2021-05-14T15:26:23" hostname="holmans-iMac.local" time="32.537">
  <properties/>
  <testcase name="login" classname="com.segurosbolivar.automation.tests.login.RunnerLogin" time="32.537">
    <failure message="java.lang.AssertionError: expected [asdf] but found [HOLA PRUEBAS]" type="java.lang.AssertionError">java.lang.AssertionError: expected [asdf] but found [HOLA PRUEBAS]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at com.segurosbolivar.automation.tests.login.StepsLogin.fillAll(StepsLogin.java:32)
	at com.segurosbolivar.automation.tests.login.RunnerLogin.login(RunnerLogin.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <system-out><![CDATA[https://7scw0ywnt9.execute-api.us-east-1.amazonaws.com/prod/data/1/case/18
Successful Enable: ** :) **   
Current Window State       : complete
Successful Enable: ** :) **   
Successful Enable: ** :) **   
Current Window State       : complete
Successful Enable: ** :) **   
elemento//input[@id='loginInPass']
elemento//button[@id='loginUser']
Current Window State       : complete
Successful Enable: ** :) **   
]]></system-out>
  <system-err><![CDATA[Starting ChromeDriver 89.0.4389.23 (61b08ee2c50024bab004e48d2b1b083cdbdac579-refs/branch-heads/4389@{#294}) on port 25382
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
[1621005917.081][WARNING]: This version of ChromeDriver has not been tested with Chrome version 90.
May 14, 2021 10:25:17 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
]]></system-err>
</testsuite>
